modules/sdk/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties

```diff
@@ -1,17 +1,113 @@
-#liferay.workspace.bundle.token.email.address=
-#liferay.workspace.bundle.token.force=false
-#liferay.workspace.bundle.token.password=
-#liferay.workspace.bundle.token.password.file=
-#liferay.workspace.bundle.url=https://cdn.lfrs.sl/releases.liferay.com/portal/7.0.3-ga4/liferay-ce-portal-tomcat-7.0-ga4-20170613175008905.zip
-#liferay.workspace.default.repository.enabled=false
-#liferay.workspace.environment=local
-#liferay.workspace.home.dir=bundles
-#liferay.workspace.modules.default.repository.enabled=false
-#liferay.workspace.modules.dir=modules
-#liferay.workspace.plugins.sdk.dir=plugins-sdk
-#liferay.workspace.themes.dir=themes
-#liferay.workspace.wars.dir=wars
-
-microsoft.translator.client.id=
-microsoft.translator.client.secret=
\ No newline at end of file
+##
+## Liferay Workspace
+##
+
+    #
+    # Set this to true if the "liferay.workspace.bundle.url" property is set to
+    # a DXP bundle Zip. This property allows the token residing in the
+    # "~/.liferay" folder to be used to validate your user credentials when
+    # downloading the bundle. The default value is false.
+    #
+    #liferay.workspace.bundle.token.download=false
+
+    #
+    # Set the email address to use when downloading a DXP bundle. This must
+    # match the email address registered for your DXP subscription. Only use
+    # this property if you do not want to use a workspace token to supply your
+    # user credentials (i.e., liferay.workspace.bundle.token.download=false).
+    #
+    #liferay.workspace.bundle.token.email.address=
+
+    #
+    # Set this to true to override the existing token with a newly generated
+    # token created by the "createToken" task. The default value is false.
+    #
+    #liferay.workspace.bundle.token.force=false
+
+    #
+    # Set the password to use when downloading a DXP bundle. This must match the
+    # password registered for your DXP subscription. Only use this property if
+    # you do not want to use a workspace token to supply your user credentials
+    # (i.e., liferay.workspace.bundle.token.download=false).
+    #
+    #liferay.workspace.bundle.token.password=
+
+    #
+    # Set the file to hold the Liferay bundle authentication token password.
+    # The default file value is "~/.liferay/token".
+    #
+    #liferay.workspace.bundle.token.password.file=
+
+    #
+    # Set the URL pointing to the bundle Zip to download. If the URL points to a
+    # DXP bundle (e.g., https://api.liferay.com/...), set the
+    # "liferay.workspace.bundle.token.download" property to true. The default
+    # value is the URL for Liferay Portal CE 7.0.3 GA4.
+    #
+    #liferay.workspace.bundle.url=https://cdn.lfrs.sl/releases.liferay.com/portal/7.0.3-ga4/liferay-ce-portal-tomcat-7.0-ga4-20170613175008905.zip
+
+    #
+    # Set this to true to configure Liferay CDN as the default repository in the
+    # root project. The default value is false.
+    #
+    #liferay.workspace.default.repository.enabled=false
```

Comment:
If the default value is `true` here, should we change the property to reflect that as well?